pkg/antctl/command_definition.go

```diff
@@ -127,6 +127,14 @@ func (e *resourceEndpoint) flags() []flagInfo {
 			usage:        "Filter the resource by namespace",
 		})
 	}
+	if e.groupVersionResource == &v1beta2.NetworkPolicyVersionResource {
+		flags = append(flags, flagInfo{
+			name:         "sort-by",
+			shorthand:    "b",
```

Comment:
curious why "b" ? i don't think kubectl has any short-hand for it either

Reply:
I was thinking "b" to be short-hand for "by" but you're right we don't necessarily need to have a short hand for this. Removed.